.github/workflows/cont_integration.yml

```diff
@@ -38,7 +38,9 @@ jobs:
           cargo update -p flate2:1.0.27 --precise "1.0.26"
           cargo update -p reqwest --precise "0.11.18"
           cargo update -p h2 --precise "0.3.20"
-          cargo update -p rustls-webpki --precise "0.100.1"
+          cargo update -p rustls-webpki:0.100.3 --precise "0.100.1"
+          cargo update -p rustls-webpki:0.101.5 --precise "0.101.1"
```

Comment:
Why do we depend on two slightly different versions of webpki?

Reply:
Looks like `rustls-webpki` 0.100.x is required by the `rustls` version used by `electrum-client`. 

And `rustls-webpki` 0.101.x is required directly by the version of `minreq` used by our dev-dependencies `bitcoind` and `electrsd`. 

I think the ultimate solution for this will be to get all our dependencies to use the same major+minor version of `rustls`.